app/controllers/merchant_items_controller.rb

```diff
@@ -1,4 +1,26 @@
 class MerchantItemsController < ApplicationController
   def index
+    @merchant = Merchant.find(params[:merchant_id])
   end
+
+  def show
+    @item = Item.find(params[:item_id])
+  end
+
+  def new
+    @merchant = Merchant.find(params[:merchant_id])
+    @item = Item.new
+  end
+
+  def create
+    params[:unit_price] = (params[:unit_price].to_f * 100).to_i
+    merchant = Merchant.find(params[:merchant_id])
+    item = merchant.items.create(item_params)
+    redirect_to "/merchants/#{params[:merchant_id]}/items"
+  end
```

Comment:
We talked about this already but we might want to revisit this during refactoring to see if there is a better way to do the conversion than in the create method (or maybe just a DRYer way within this method). It gets the job done though  😀